go.mod

```diff
@@ -1,9 +1,13 @@
 module github.com/derision-test/glock
 
-go 1.15
+go 1.18
```

Comment:
Updated to 1.18 so we can use generics for some of the channel test utility functions.